internal/cmd/pipeline/command_create.go

```diff
@@ -137,8 +133,17 @@ func (c *command) create(cmd *cobra.Command, _ []string) error {
 		createPipeline.Spec.KsqlCluster = &streamdesignerv1.ObjectReference{Id: ksqlCluster}
 	}
 
-	// add Schema Registry Cluster if its provisioned
-	if _, ok := srCluster.GetIdOk(); ok {
+	// check if we need to configure Schema Registry cluster for this pipeline
+	useSchemaRegistry, err := cmd.Flags().GetBool("use-schema-registry")
+	if err != nil {
+		return err
+	}
+
+	if useSchemaRegistry {
+		srCluster, err := c.Context.FetchSchemaRegistryByEnvironmentId(environmentId)
+		if err != nil {
+			return err
```

Comment:
what would be the error message here

Reply:
Not sure on the specific text, but if the Schema Registry cluster hasn't been enabled, I think you'd see a corresponding error here.